package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "start": "node ./server/bin.js",
-    "test": "grunt test"
+    "test": "grunt test",
+    "postinstall": "./node_modules/bower/bin/bower install"
```

Comment:
No need, I can move it to the `Procfile` and just run `bower install && grunt build && node ./server/bin.js`, once the above issue is taken care of. As of now it is removed from `package.json`.